src/ec/packages/ec-component-page-banner/page-banner.story.js

```diff
@@ -68,22 +164,89 @@ storiesOf('Components/Banners/Page Banner', module)
   .addDecorator(withNotes)
   .addDecorator(withCode)
   .addDecorator(withKnobs)
-  .add('image', () => pageBanner(preparePageBanner(bannerDataImage)), {
-    notes: { markdown: notes, json: bannerDataImage },
-  })
+  .add(
+    'image',
+    () => {
+      button('With or without title', imageTitleBtnToggler, tabLabels.cases);
+      button('With or without baseline', imageDescBtnToggler, tabLabels.cases);
+      button('With or without cta', imageCtaBtnToggler, tabLabels.cases);
+      button('Reset the layout', imageResetBtnToggler, tabLabels.cases);
+
+      return pageBanner(preparePageBanner(imageData));
+    },
+    {
+      notes: { markdown: notes, json: imageData },
+    }
+  )
   .add(
     'image-shade',
-    () => pageBanner(preparePageBanner(bannerDataImageShade)),
+    () => {
+      button(
+        'With or without title',
+        imageShadeTitleBtnToggler,
+        tabLabels.cases
+      );
+      button(
+        'With or without baseline',
+        imageShadeDescBtnToggler,
+        tabLabels.cases
+      );
+      button('With or without cta', imageShadeCtaBtnToggler, tabLabels.cases);
```

Comment:
"cta" by itself doesn't mean much. Could be CTA, or cta link, cta button or whatever, otherwise no context.

Reply:
i renamed it to "call to action"